### DISCUSSION TITLE
Discontinuous finite element

### DISCUSSION THREAD

**Discussion by nanguaxiaofendui** [libMesh/libmesh#3635] [Q&A]
Does libmesh contain intermittent finite element? If so, how to use it? Please help me, thank you.

**Comment by roystgnr** [comment]
I'm not familiar with "intermittent finite element" methods, I'm afraid.

libMesh does have a few discontinuous finite element families: `MONOMIAL` (P^n polynomials in master element space), `XYZ` (P^n in physical space, zeroed at the element center), `L2_LAGRANGE` (nodal Lagrange, so e.g. Q^n polynomials on quads and hexes vs P^n on triangles and tetrahedra, but discontinuous between elements rather than continuous as in `LAGRANGE`), and `L2_HIERARCHIC` (the same family of spaces, but with a change of basis so it's the discontinuous version of `HIERARCHIC` instead).
